.storybook/webpack.config.js

```diff
@@ -0,0 +1,26 @@
+// module.exports = {
+//   module: {
+//     rules: [
+//       {
+//         test: /\.css$/,
+//         use: [
+//           { loader: 'style-loader' },
+//           {
+//             loader: 'css-loader',
+//             options: {
+//               modules: true
+//             },
+//           },
+//         ],
+//       },
+//     ],
+//   },
+// };
```

Comment:
This commented out config should work... and used to in Storybook 4. It doesn't right now, though, hence the nonsense below.